forc-plugins/forc-client/src/util/encode.rs

```diff
@@ -1,47 +1,11 @@
 use std::str::FromStr;
 
 use fuel_abi_types::abi::full_program::FullTypeApplication;
-use serde::{Deserialize, Deserializer, Serialize};
 
 /// A wrapper around fuels_core::types::Token, which enables serde de/serialization.
-#[derive(Debug, PartialEq)]
+#[derive(Debug, PartialEq, serde::Serialize, serde::Deserialize)]
 pub(crate) struct Token(pub(crate) fuels_core::types::Token);
 
```

Comment:
I think we can drop the wrapper `Token` since underlying `Token` now implements `Serialize` and `Deserialize`. Just leaving this as a comment, no need to do that here i can tackle that later on 👍